### ISSUE TITLE
Add option to sanitize data

### ISSUE THREAD

**Issue by bdbch** [bdbch/js-state-handler#9]
It would be great if there would be an option which allows data to be sanitized. Either it's defined on the initial state created on the constructor or it's done via another set function.

Examples:

```js
var State = new StateHandler({
  apple: 4,
  name: {
    sanitize: true,
    value: ''
  }
})
```

or

```js
var State = new StateHandler({
  name: ''
})

State.set({
  name: '<script>// malicious code</script>'
}, {
  sanitized: true
})
```

**Comment by rivergillis** [comment]
Hi, so as far as actually sanitizing the data, would that just be a matter of removing certain punctuation and replacing them with the HTML entities? Sort of like `escape()` from [validator.js](https://github.com/chriso/validator.js).

**Comment by bdbch** [comment]
@rivergillis yeah exactly. It should also strip out html tags so you can't add malicious code to a website handling states.
